### ISSUE TITLE
AE Just stopped working on all versions

### ISSUE THREAD

**Issue by NeuroSMP** [GC-spigot/AdvancedEnchantments#4087]
### Describe the bug

Can no longer use AE/ nothing as far as i know has changed on the server plugin list.
(/ae plinfo skipPlugins) wont work either.

### How to reproduce

/gkit and any other /command, 

### "/ae plinfo" link

/ae plinfo skipPlugins ... sorry wont work

### Server Log

https://mcpaste.io/df5896e00456449d

**Comment by NeuroSMP** [comment]
Just tried (AdvancedEnchantments-9.5.29) and that works perfectly.

**Comment by NeuroSMP** [comment]
Caused by: java.lang.IllegalStateException: zip file closed

**Comment by ThomasWega** [comment]
Have you tried letting it recreate your default files?

**Comment by NeuroSMP** [comment]
Yeah, i have deleted all AE files off the server but still not working.

**Comment by NeuroSMP** [comment]
> AdvancedEnchantments

can you see any conflicts please? 

![image](https://github.com/GC-spigot/AdvancedEnchantments/assets/163584065/cba47f67-2457-48f5-88e3-77792f714951)

**Comment by ThomasWega** [comment]
What version are you on?

**Comment by GC-spigot** [comment]
This error occurs when something is messing with plugin loading, e.g.:
- reloading
- plugman (or other dynamic loading plugins)
- corrupted file
Check if you do not have duplicate AE files in plugins folder, and redownload the plugin :)
